src/query/ast/tests/it/testdata/statement.txt

```diff
@@ -6975,6 +6975,163 @@ Copy(
 )
 
 
+---------- Input ----------
+COPY INTO mytable
+                FROM (SELECT t.c1
+                    FROM @mystage/path/to/file (FILE_FORMAT=> 'PARQUET' ) t
+                    where t.c2 > 0
+                );
+---------- Output ---------
+COPY INTO mytable FROM (SELECT t.c1 FROM @mystage/path/to/file (FILE_FORMAT => 'PARQUET') AS t WHERE (t.c2 > 0)) SINGLE = false PURGE = false FORCE = false ON_ERROR = 'abort'
```

Comment:
select `FILE_FORMAT => 'PARQUET'` and copy `FILE_FORMAT = PARQUET` mixed here is confusing, snowflake does not do that, it does not have the options for the SELECT, any better solution?

Reply:
`(SELECT t.c1 FROM @mystage/path/to/file (FILE_FORMAT => 'PARQUET')`
in snow will use the stage file option:
`(SELECT t.c1 FROM @mystage/path/to/file)`

Reply:
hmm, no need forbidden, let user to do that is ok, like snow did, in his example as above, it doesn't have the option to confuse the user who want transforme the data from a stage.

Reply:
what about only report error when the same option is specified in both places and is diff?

```
COPY INTO mytable FROM (SELECT t.c1 FROM @mystage/path/to/file(FILE_FORMAT => 'CSV') AS t WHERE (t.c2 > 0)) SINGLE = false PURGE = false FORCE = false ON_ERROR = 'abort' FILE_FORMAT = 'PARQUET'
```